gateway/src/main/resources/application.yml

```
@@ -3,4 +3,4 @@ spring:
     name: gateway-service
 
   config:
-    import: configserver:${CONFIG_SERVER_URL:http://localhost:19000}
\ No newline at end of file
+    import: configserver:http://localhost:19000
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)